piplmesh/panels/bicikelj/panel.py

```
@@ -0,0 +1,35 @@
+from __future__ import absolute_import
+from django.utils.translation import ugettext_lazy as _
+from piplmesh import panels
+from . import stations
+
+class BicikeljPanel(panels.BasePanel):
+    def get_context(self, context):
+
+        context.update({
+            'header': _("Bicikelj stations"),
+        })
+
+        try:
+            node = self.request.node
+        except :
```

This is not possible. Furthermore, never catch ALL exceptions, only those you expect.